src/System.Data.Common/tests/System/Data/DataTableReadXmlSchemaTest.cs

```diff
@@ -422,5 +423,274 @@ public void ReadConstraints()
             Assert.Equal(0, ds.Tables[1].Constraints.Count);
             Assert.Equal("Constraint1", ds.Tables[0].Constraints[0].ConstraintName);
         }
+
+        [Fact]
+        public void XsdSchemaSerializationIgnoresLocale()
+        {
+            RemoteInvoke(() =>
+            {
+                var serializer = new System.Runtime.Serialization.Formatters.Binary.BinaryFormatter();
+
+                var table = new DataTable();
+
+                table.Columns.Add(
+                    new DataColumn("RowID", typeof(int))
+                    {
+                        AutoIncrement = true,
+                        AutoIncrementSeed = -1, // These lines produce attributes within the schema portion of the underlying XML representation of the DataTable with the values "-1" and "-2".
+                        AutoIncrementStep = -2,
+                    });
+
+                table.Columns.Add("Value", typeof(string));
+
+                table.Rows.Add(1, "Test");
+                table.Rows.Add(2, "Data");
+
+                var buffer = new MemoryStream();
+
+                var savedCulture = CultureInfo.CurrentCulture;
+
+                try
+                {
+                    // Before serializing, update the culture to use a weird negative number format. This test is ensuring that this is ignored.
+                    CultureInfo.CurrentCulture =
+                        new CultureInfo("en-US")
+                        {
+                            NumberFormat =
+                                new NumberFormatInfo()
```

Comment:
merge line 459 & 460. I will stop commenting on these. It's just about our code style and about readability. Breaking declarations into multiple lines makes it harder to read.